tests/integration/api_swarm_test.py

```diff
@@ -45,7 +45,8 @@ def test_init_already_in_cluster(self):
     @requires_api_version('1.24')
     def test_init_swarm_custom_raft_spec(self):
         spec = self.client.create_swarm_spec(
-            snapshot_interval=5000, log_entries_for_slow_followers=1200
+            name='default', snapshot_interval=5000,
```

Comment:
Is this change needed? I think moby already sets the default if no name is specified; https://github.com/moby/moby/blob/47b3209f687c240dc2841bcfc147b2ffa341a7fe/daemon/cluster/swarm.go#L460-L464

Reply:
You're right. Thanks for pointing this out! @thaJeztah 


The call to init_swarm() below will sanitize it and add the name if its not added.

I didn't recall this and made the change everywhere. I will remove it and only keep it only for test_update_swarm() where init_swarm() is not called.